.github/workflows/test.yml

```diff
@@ -50,6 +50,8 @@ jobs:
         run: |
           if [ "${{matrix.lower_bounds}}" ]
           then
+            pip install "cython<3.0.0" wheel
+            pip install "pyyaml==5.4.1" --no-build-isolation
```

Comment:
Don't do multiple "pip install" if you can avoid it.

Reply:
I am sorry but I cannot go past the installation step for "pulp-cli CI / test / test (3.11, latest, true) (pull_request) " because of the following issue: https://github.com/yaml/pyyaml/issues/601. Furthermore, I cannot install updated `pyyaml==6.0.1` because there is a lowerbound in `pulp-cli==0.20` that requires `pyyaml==5.4.1`.